deb-exclude

```diff
@@ -0,0 +1,10 @@
+.git
+.github
+.gitignore
+.gitattributes
+.travis.yml
+appveyor.yml
+DEBIAN
+refine.bat
+deb-exclude
+deb-creation.sh
```

Comment:
Could we move this file to `.deb-exclude` maybe, so that it is hidden by default, just like `.gitignore`?